doc/api/core/pointsets.rst

```diff
@@ -63,6 +76,7 @@ classes inherit all methods from their corresponding VTK
    :toctree: _autosummary
```

Comment:
Can't comment above this line, but we might want to also add `vtkPointSet` to the paragraph right above this line.

And I wonder if we should also add `PointSet` to `api/core/index.rst`.

Reply:
Agreed. Done in c3b8e089f23ed31b275201d8f08e69732ec611b0.